pulseaudio-control.bash

```diff
@@ -300,8 +320,16 @@ function output() {
 
 
 function usage() {
-    echo "Usage: $0 ACTION"
-    echo ""
+    echo "Usage: $0 [OPTION...] ACTION"
+    echo
+    echo "Options:"
+    echo "    --vol-icons       icons for volume, from lower to higher (comma-separated)"
+    echo "    --vol-icon-mute   icon to use when muted"
+    echo "    --sink-blacklist  sinks to ignore when switching (comma-separated)"
+    echo "    --sink-icon       icon to use for sink"
+    echo "    --sink-name-from  pacmd property to use for sink name"
+    echo "    --sink-nickname   <name>:<nick> pair to use for sink name (multiple args allowed)"
```

Comment:
One last thing: if the option takes a value it should be indicated clearly in the `--help` message. For example, `--vol-icons` should be `--vol-icons ICONS` or with a `=` as the delimiter.

Reply:
Oops yes, thanks.